src/Types/TokenStatusEnum.php

```diff
@@ -0,0 +1,81 @@
+<?php declare(strict_types=1);
+
+namespace App\Users\GraphQL\Types;
+
+use GraphQL\Type\Definition\EnumType;
+
+class TokenStatusEnum extends EnumType
+{
+    /**
+     * Valid token
+     */
+    const STATUS_OK = 'OK';
+
+    /**
+     * Not a valid token
+     */
+    const STATUS_INVALID = 'INVALID';
+
+    /**
+     * Expired but can be renewed
+     */
+    const STATUS_EXPIRED = 'EXPIRED';
+
+    /**
+     * Expired and cannot be renewed
+     */
+    const STATUS_DEAD = 'DEAD';
+
+    /**
+     * Provided user / password were incorrect
+     */
+    const STATUS_BAD_LOGIN = 'BAD_LOGIN';
+
+    public function __construct()
+    {
+        $values = [
```

Comment:
Could you extract this in to a static from config? It doesn't look like it needs to be written fully in the construct?